.github/workflows/codeql.yml

```diff
@@ -0,0 +1,60 @@
+
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ develop ]
+  schedule:
+    - cron: '37 8 * * 6'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript', 'python' ]
```

Comment:
Let’s disable python by default on the codeql.

Reply:
I have disabled Python, and It was the default when I enabled that GitHub suggested that and I kept it as is.